src/components/NavigationTabs/index.tsx

```diff
@@ -39,15 +39,17 @@ const StyledArrowLeft = styled(ArrowLeft)`
 `
 
 const ButtonBack = styled(ButtonEmpty)`
-  width: 36px;
-  height: 36px;
+  // TODO namgold: handle this to handle 2 case old and new
```

Comment:
Because `ref= useRef()` has some delay to set the ref value. When it's settle done, it doesn't trigger the render again, so I need to make it a state to track whenever it's settle done to render and calc value again.